packages/@react-spectrum/list/src/styles.css

```diff
@@ -337,7 +337,7 @@
     align-items: center;
     justify-content: center;
 
-    > img, > svg {
```

Comment:
this wasn't being used because the svg isn't a child of `--thumbnail`, it has the class itself